donate.html

```diff
@@ -65,9 +65,68 @@
 
 </head>
 <body>
+
 <div class="container-fluid">
   <!-- su'ad -->
-
+  <!-- Header -->
+  <div class = "donate-header">
+    <div style="text-align:center;">
```

Comment:
you can add the bootstrap class "text-center" which centers things inside the div instead of manually adding styling to center stuff!